File: SourceCode/Debug Helpers/open.fda.gov/ApiDalc/Enumerations/DrugCharacterization.vb
Imports System.ComponentModel

Namespace Enumerations

    ''' <summary>
    ''' Reported role of the drug in the adverse event.
    ''' </summary>
    ''' <remarks></remarks>
    Public Enum DrugCharacterization

        <Description("Suspect drug")>
        SuspectDrug = 1

        <Description("Concomitant drug")>
        ConcomitantDrug = 2

        <Description("Interacting drug")>
        InteractingDrug = 3

    End Enum

End Namespace

File: SourceCode/AspCoreVersion/src/ShopAware.Core/Enumerations/DrugCharacterization.cs
using ShopAware.Core.Attributes;

namespace ShopAware.Core.Enumerations
{
    /// <summary>
    ///     Reported role of the drug in the adverse event.
    /// </summary>
    /// <remarks></remarks>
    public enum DrugCharacterization
    {
        [Description("Suspect drug")] SuspectDrug = 1,

        [Description("Concomitant drug")] ConcomitantDrug = 2,

        [Description("Interacting drug")] InteractingDrug = 3
    }
}